### DISCUSSION TITLE
Removing original files manually ?

### DISCUSSION THREAD

**Discussion by shodanx2** [mediacms-io/mediacms#563] [General]
Hi,

I have a quite small mediacms server (only 40gb) and I'm almost full. 

I would like to delete the original files but I'm not sure how to do it properly

I see that the original files are all stored in 

/opt/mediacms/media_files/original/user/myusername/

And I could easily rm * them all but I suspect mediacms will then break when it tried to access those files ?

I have downloading of videos disabled, and I think downloading and encoding are the only times when the originals are access by mediacms.

So maybe it is safe to just delete those files ?

Another option might be to delete the "media"  in https://mediacms.example.com/admin/files/media/ ?

But I'm not sure if that's the video post on mediacms, or if that's just a handle for the original file. 

I checked in /admin/files/encoding/, but there are no mentions of the originals in that place

**Comment by mgogoulos** [comment]
Hi, you are correct, the original file is used for transcoding all versions and after that it's available for download (as part of the download options).  Don't delete it from admin, because this will remove the Media object (so not only the file but also encodings and the media page won't exist any more). 

The correct way to do this would be to introduce a new setting (delete_files_after_encoding) and if enabled have the original media deleted after processing, this is definitely one of the issues I'm planning to work as soon as I have some time. Until then, try to remove a media from the path (on the file path you have identified) and see whether this creates any issue (other than not being able to download the original file).

**Reply by Unicorn-Zombie-Apocalypse** [reply]
Hello!
Could you tell if the feature was implemented?

**Reply by johnkraczek** [reply]
I am also curious if this was implemented.
